internal/pkg/util/image/ocisif.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2023, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package image
```

Comment:
I don't see a good reason not to have this as a public convenience function in `pkg/image`?

I don't think it is at risk of an API change.